transport/src/main/java/me/uport/sdk/transport/Transports.kt

```diff
@@ -11,22 +13,44 @@ class Transports {
      *  Sends a selective disclosure request to the uPort app.
      *  If the uPort app is not present this will open a browser
      *
-     * __API-maturity: new__
+     * __API volatility: high__
      *
      *  This method does not ensure any type of response.
      */
     fun send(context: Context, jwt: String) {
-        val encodedQuery = URLEncoder.encode(jwt, "UTF-8")
-        val uri = Uri.parse("https://id.uport.me/req/$encodedQuery?callback_type=redirect")
-
-
-        //deprecated calling convention
-//        val uri = Uri.parse("me.uport:me?requestToken=$encodedQuery")
-
+        val uri = encodeURICall(jwt)
 
         val intent = Intent(Intent.ACTION_VIEW, uri)
                 .addCategory(Intent.CATEGORY_BROWSABLE)
+
         context.startActivity(intent)
     }
 
+    /**
+     *  Sends a selective disclosure request to the uPort app.
```

Comment:
This comment is misleading, the method does not send only selective disclosure requests

Reply:
good catch